core/validation/pre_transformation_validate.py

```diff
@@ -0,0 +1,148 @@
+import pandas as pd
+
+import core.validation.failures as vf
+from core.const import (
+    FUND_TYPE_TO_TD_OR_HS,
+    GET_FORM_VERSION_AND_REPORTING_PERIOD,
+    PLACE_TO_FUND_TYPE,
+    TF_PLACE_NAMES_TO_ORGANISATIONS,
+)
+from core.exceptions import ValidationError
+
+
+def pre_transformation_validations(
+    workbook: dict[str, pd.DataFrame], reporting_round: int, place_names: tuple, fund_types: tuple
+) -> list[vf.ValidationFailure] | None:
+    """
+    Perform pre-transformation validations on the given workbook for the specified reporting round.
+
+    :param workbook: A dictionary containing sheets of the workbook, where keys are sheet names and
+                     values are pandas DataFrames representing each sheet in the submission.
+    :param reporting_round: An integer representing the reporting round for the validation.
+    :param place_names: A tuple of place names for validation.
+    :param fund_types: A tuple of fund types for validation.
+    :return: A list of ValidationFailure objects if any validation failures occur, or None if no errors are found.
+    """
+    if reporting_round in [1, 2]:
+        return  # do not do pre-transformation validation on historical data
+
+    if reporting_round == 4:
+        authorisation_validation(workbook, place_names, fund_types)  # return first as unauthorised cannot be validated
+    wrong_input_validation(workbook, reporting_round)
+    conflicting_input_validation(workbook)  # return last as invalid input should not be
+    # returned with conflicting input
+
+
+def authorisation_validation(workbook: dict[str, pd.DataFrame], place_names: tuple, fund_types: tuple):
+    """
+    Validate authorisations for the given workbook.
+
+    This function checks if the submitted workbook has the required authorisation for place names and fund types.
+
+    :param workbook: A dictionary containing sheets of the workbook.
+    :param place_names: A tuple of place names for validation.
+    :param fund_types: A tuple of fund types for validation.
+    :raises ValidationError: If authorisation validation fails, raises a ValidationError with associated failures.
+    """
+    authorisation_checks = {}
+    sheet = workbook.get("2 - Project Admin")
+
+    if str(sheet.iloc[6][4]).strip() in TF_PLACE_NAMES_TO_ORGANISATIONS.keys():
+        authorisation_checks["Place Name"] = (
+            str(sheet.iloc[6][4]).strip(),
+            place_names,
+        )
+    if FUND_TYPE_TO_TD_OR_HS.get(sheet.iloc[5][4], sheet.iloc[5][4]) in {"TD", "HS"}:
+        authorisation_checks["Fund Type"] = (
+            FUND_TYPE_TO_TD_OR_HS.get(sheet.iloc[5][4], sheet.iloc[5][4]),
+            fund_types,
+        )
```

Comment:
This above may be harder to achieve here as there is a bit more logic.. there may be a way though